### PR TITLE
area_dro

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 8 functions (+0.09%) on `us`.

⚠️ This PR introduces 6 warnings:
- src/world/common/entity/Pipe.inc.h:3: warning: redundant redeclaration of `dro_01_Entity_Pipe_GetCameraYaw' in same scope
- src/world/common/entity/Pipe.inc.h:3: warning: redundant redeclaration of `dro_01_Pipe_GetCameraYaw' in same scope
- src/world/common/entity/Pipe.inc.h:4: warning: redundant redeclaration of `dro_01_Entity_Pipe_GetCurrentFloor' in same scope
- src/world/common/entity/Pipe.inc.h:4: warning: redundant redeclaration of `dro_01_Pipe_GetCurrentFloor' in same scope
- src/world/common/entity/Pipe.inc.h:5: warning: redundant redeclaration of `dro_01_Entity_Pipe_GetEntryPos' in same scope
- src/world/common/entity/Pipe.inc.h:5: warning: redundant redeclaration of `dro_01_Pipe_GetEntryPos' in same scope
- src/world/common/entity/Pipe.inc.h:6: warning: redundant redeclaration of `dro_01_Entity_Pipe_SetAnimFlag' in same scope
- src/world/common/entity/Pipe.inc.h:6: warning: redundant redeclaration of `dro_01_Pipe_SetAnimFlag' in same scope
- src/world/common/entity/Pipe.inc.h:7: warning: redundant redeclaration of `dro_01_Entity_Pipe_GetPointAheadOfPlayer' in same scope
- src/world/common/entity/Pipe.inc.h:7: warning: redundant redeclaration of `dro_01_Pipe_GetPointAheadOfPlayer' in same scope
- src/world/common/entity/Pipe.inc.h:8: warning: redundant redeclaration of `dro_01_Entity_Pipe_AwaitDownInput' in same scope
- src/world/common/entity/Pipe.inc.h:8: warning: redundant redeclaration of `dro_01_Pipe_AwaitDownInput' in same scope

**Review by ethteck — APPROVED** [review]

File: include/enums.h [hunk 1]
```diff
@@ -4832,7 +4841,7 @@ enum EffectInstanceFlags {
     EFFECT_INSTANCE_FLAGS_2                 = 0x00000002,
     EFFECT_INSTANCE_FLAGS_4                 = 0x00000004,
     EFFECT_INSTANCE_FLAGS_8                 = 0x00000008,
-    EFFECT_INSTANCE_FLAGS_10                = 0x00000010,
+    EFFECT_INSTANCE_FLAGS_10                = 0x00000010, // mark ready for cleanup
```
Comment (nanaian) [inline include/enums.h:4844]: worth naming the actual flag?

Comment (HailSanta) [reply include/enums.h:4844]: I suppose so. I was going to wait for the function managing their lifecycle to be decomped.

Comment (nanaian) [reply include/enums.h:4844]: fine with me! :)

**Review by nanaian — APPROVED** [review]

**Pull request merged**